### DISCUSSION TITLE
Github project notification

### DISCUSSION THREAD

**Discussion by pondkm** [community/community#112197] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Recently, I have tried to use Github project for task planning and I have a problem about notification setting. I want notification every time that I updates anything on the project. How can I set notification for that?

**Comment by gerteck** [comment]
To receive notifications for updates on a GitHub project, you may want to adjust your notification settings for the related specific repository.

* Navigate to the Repository: Go to the GitHub repository for your project.
* Click on the "Watch" Button: Near the top-right corner of the repository page, you'll find a button labeled "Watch" with an eye icon. Click on it. (Select All Activity)

Hope this helps in some way!

**Reply by Macolc27** [reply]
@Macolc27

**Reply by pondkm** [reply]
@gerteck  Thank you for your answer but I mean Github project as attached image. I didn't see the "Watch" button here. 
<img width="1407" alt="Screenshot 2024-03-12 at 7 02 12 PM" src="https://github.com/community/community/assets/23046294/f2738ebd-5370-416a-96c5-de4c8236bf3b">

**Reply by gerteck** [reply, marked as answer]
@pondkm 
As per their [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects), since GitHub Projects is designed to work with repositories, most notifications are tagged to a specific repository (e.g. PRs, Issues). I am not sure if it has standalone notification features. 

> A project is an adaptable spreadsheet, task-board, and road map that integrates with your **issues and pull requests** on GitHub to help you plan and track your work effectively. Your projects are built from the issues and pull requests you add, creating direct references between your project and your work. 

Perhaps if you are looking for a standalone task manager / Kanban Board etc, you might want to explore other options? (E.g. Notion, MS Teams)

**Reply by pondkm** [reply]
@gerteck Got it. Thank you very much!
